modules/openideal_statistics/src/Plugin/Block/OpenidealStatisticsIdeaStatisticsBlock.php

```diff
@@ -54,30 +55,31 @@ public function build() {
     $theme_path = base_path() . $this->themeManager->getActiveTheme()->getPath();
     $build['#theme'] = 'site_wide_statistics_block';
     $build['#main_class'] = 'idea-statistics-block';
+    $build['#show_title'] = !$public_stream;
     $build['#content'] = [
       'votes' => [
         'bottom' => [
           '#lazy_builder' => ['openideal_statistics.lazy_builder:getVotes', [$id]],
           '#create_placeholder' => TRUE,
         ],
         'title' => $this->t('Votes'),
-        'img' => $theme_path . '/misc/icons/like_tag.svg',
+        'img' => $theme_path . '/misc/icons/' . ($public_stream ? 'public_stream_like' : 'like_tag') . '.svg',
```

Comment:
Can we pass just CSS classes and use it to on twig template to apply images?
It should be a more convenient way, isn't it?

Reply:
But it's SVG images, and it hard to add them via CSS, or do you meant to add them in another way?

Reply:
Nevermind.